### ISSUE TITLE
Error for PCL v4.5 Profile 7 + Newtonsoft.Json

### ISSUE THREAD

**Issue by anfomin** [aspnet/dnx#2182]
When I added Newtonsoft.Json to the PCL profile 7 I got compile error:
> The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'mscorlib, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e, Retargetable=Yes'.

Here is `project.json`:
```son
{
	"dependencies": {
		"Newtonsoft.Json": "7.0.1"
	},

	"frameworks": {
		".NETPortable,Version=v4.5,Profile=Profile7": {
			"frameworkAssemblies": {
				"System.Runtime": ""
			}
		}
	}
}
```

Sample: https://github.com/anfomin/DNX-PCL-test/tree/master/PCL7

**Comment by muratg** [comment]
@anurse

**Comment by muratg** [comment]
Closing this as @anurse seems to be tracking it in #2180

**Comment by anurse** [comment]
Confirmed that the fix for #2180 fixes this issue as well.
